apps/web/stylesheets/style.css

```diff
@@ -0,0 +1,130 @@
+body {
+  color: #000;
+  font-family: 'Dekko', cursive;
+  font-size: 15px;
+  margin: 0px;
+  padding: 0px;
+  background-color: #fff;
+}
+
+.main-menu-wrapper {
+  width: auto;
+  height: 90vh;
+  background-image: url("https://i.imgur.com/yOluG2D.jpg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  display: flex;
+  margin-top: 5vh;
+  margin-bottom: 5vh;
+  overflow-y: hidden;
+}
+
+.entries-menu-wrapper {
+  width: 50%;
+  height: 90vh;
+  background-image: url("https://i.imgur.com/5W3ePlt.jpg");
```

Comment:
Might be a good idea to just download the images we use and serve them from public directory. Should speed up rendering significantly